x-pack/plugins/ml/public/ui_actions/get_embeddable_time_range.ts

```diff
@@ -7,9 +7,9 @@
 
 import type { TimeRange } from '@kbn/es-query';
 import { apiHasParentApi, apiPublishesTimeRange } from '@kbn/presentation-publishing';
-import type { MlEmbeddableBaseApi } from '../embeddables';
+import type { MlEmbeddableBasicApi } from '../embeddables';
 
-export const getEmbeddableTimeRange = (embeddable: MlEmbeddableBaseApi): TimeRange | undefined => {
+export const getEmbeddableTimeRange = (embeddable: MlEmbeddableBasicApi): TimeRange | undefined => {
```

Comment:
could you please revert this change

Reply:
As Devon mentioned [above](https://github.com/elastic/kibana/pull/180544#discussion_r1561770585), with the updated `fetch$` helper, we will no longer need to handle filters/query/refresh ourselves so the base type will no longer need to encompass that.

Since the swimlane embeddable still handles those properties itself, we still need the `MlEmbeddableBaseApi` for that but ideally it sounds like we want to move away from handling those ourselves if we don't need to. 

The `MlEmbeddableBasicApi` introduced in this PR removes those unnecessary types handled by the parent - this can be temporary until swimlane embeddable is updated and then they can both use the same base type.

Let me know if I'm missing something here. Also happy to rename if there's a better option to make it clearer.

Reply:
I guess we can update the base type used by the swim lane, and move filters/query/refresh to the AnomalySwimLane embeddable api. In that case we keep only one valid base type